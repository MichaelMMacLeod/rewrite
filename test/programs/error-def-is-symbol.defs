(def I_am_okay because_I_am_a_definition)
I_am_a_symbol_not_a_definition