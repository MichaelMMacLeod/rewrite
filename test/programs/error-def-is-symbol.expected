error[E009]: expected definition, found symbol
./test/programs/error-def-is-symbol.defs:1:1
   |
 1 | I_am_a_symbol_not_a_definition
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ this should be a definition
help: definitions look like this: '(def <pattern> <constructor>)'
